src/app/layout/Layout.tsx

```diff
@@ -14,14 +14,15 @@ export function Layout({ children }: { children: React.ReactNode }) {
     setSidebarOpen(!isSidebarOpen);
   };
 
+  const isRestrictPath =
+    pathname === Routes.Playground || pathname === Routes.Login;
```

Comment:
I found out another way to make this work but this would be a blocker for you. Next.js has this thing called Route Groups where each group could have layout.tsx of it's own.

https://nextjs.org/docs/app/building-your-application/routing/route-groups

No need to implement it but I did implement that on Finnodes yesterday.